core/src/main/java/com/datastax/oss/driver/api/core/metrics/DefaultSessionMetric.java

```diff
@@ -23,6 +23,9 @@
   CONNECTED_NODES("connected-nodes"),
   CQL_REQUESTS("cql-requests"),
   CQL_CLIENT_TIMEOUTS("cql-client-timeouts"),
+  THROTTLING_DELAY("throttling.delay"),
```

Comment:
I was thinking that it'd also be a nice to have a way to measure how many requests are being throttled vs. not throttled, but I think it can already be inferred from cql-requests.count - throttling.delay.count, so no need for that 👍